sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/InheritedCollidingInterfaceMembers.AfterCSharp8.cs

```diff
@@ -214,7 +214,7 @@ public interface IIndexer1
 
     public interface IIndexer2
     {
-        public string this[int i] // Rule is raised but the message is not clear
+        public string this[int i] // Indexer does not have a name, signature is used instead
         {
             get { return ""; } // Noncomliant
```

Comment:
I know this is old but that's a very interesting way to hide an issue 😄. (typo `Noncompliant`)

Reply:
We should consider it as design-pattern 😄
Issue should be raised on line 226, so I'll just remove it.